### DISCUSSION TITLE
Can't click on choice buttons

### DISCUSSION THREAD

**Discussion by Jimsbond004** [dialogic-godot/dialogic#947] [Q&A]
I have some choices in my dialogues, but I can't click on the buttons to select them with my mouse, I can only use arrows and enter to select. I can click, but clicking just selects the default/ first option. I think what's happening is that the first choice is always selected so clicking is just like pressing enter, because dragging my mouse doesn't work. I'm making a pixel game so I have to make the buttons very small, maybe that's the issue. Is this a bug or is there an option/ solution?

**Comment by mak448a** [comment]
Do you have a CanvasLayer covering your buttons somewhere?
